asistente_ladm_col/gui/supplies/dlg_missing_cobol_supplies.py

```diff
@@ -74,10 +75,11 @@ def accepted(self):
         self.save_settings(self.data_system)
 
         self.folder_path = self.txt_file_path_folder_supplies.text()
+        self.files_name = self.txt_files_name_supplies.text().strip()
```

Comment:
Change name to `self.file_names`.

Reply:
I already changed in the last commit, thanks